script/pre_commit/pre_commit.md

```diff
@@ -0,0 +1,6 @@
+# Git pre-commit
+
+### Installation
+Under peloton root directory, run `ln -s ../../script/pre_commit/pre-commit.sh .git/hooks/pre-commit` for once to install locally.
```

Comment:
Though it is weird, actually I tested that the first path is relative to `.git/hooks/` (where symlink resides) and second path is relative to the current working path (the root).